doc/releases/changelog-dev.md

```diff
@@ -21,12 +21,19 @@
 * `qml.draw` and `qml.draw_mpl` now render operator ids.
   [(#4749)](https://github.com/PennyLaneAI/pennylane/pull/4749)
 
+* Added `ops.functions.assert_valid` for checking if an `Operator` class is defined correctly.
+  [(#4764)](https://github.com/PennyLaneAI/pennylane/pull/4764)
+
 <h3>Breaking changes 💔</h3>
 
 * The `prep` keyword argument has been removed from `QuantumScript` and `QuantumTape`.
   `StatePrepBase` operations should be placed at the beginning of the `ops` list instead.
   [(#4756)](https://github.com/PennyLaneAI/pennylane/pull/4756)
 
+* The matrix for the `GroverOperator` is now the negative of what it was before. The matrix
+  now matches the decomposition.
+  [(#4764)](https://github.com/PennyLaneAI/pennylane/pull/4764)
```

Comment:
Is this a merge artifact or something you changed in this PR?

Reply:
Something changed in this PR.  In testing, I found that the matrix and decomposition of `GroverOperator` differed by a global phase. I thought it better to apply that phase to the `matrix` then add a `GlobalPhase` operator to the decomposition.